Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.Shared/Controls/BocListUserControl.ascx.cs

```diff
@@ -95,6 +97,10 @@ protected override void OnPreRender (EventArgs e)
     {
       base.OnPreRender(e);
       SetTestOutput();
+
+      ((SmartPage)Page)!.PrepareValidation();
+      CurrentObjectValidationResultDispatchingValidator.DispatchValidationFailures(_validationResult);
+      CurrentObjectValidationResultDispatchingValidator.Validate();
```

Comment:
Shouldn't this be the same conditional logic as in IndividualControlTests or similiarily only get triggered when performing validation?

Reply:
The main difference here is that we have our own BocList control that has the validation column added. The checkbox stuff in the IndividualControlTests was mainly because this is only for manual clicking and introducing a new BocList seems unnecessary. In the case of the web tests it is kind of strange to enable the functionality via checkbox even if we have our own control for the validation scenario. The other BocList controls should ignore the validation anyways.

Reply:
Hmm...so we're using the CurrentObjectValidationResultDispatchingValidator which applies to all BO-Controls on the page, knowing that it's only relevant to one of them so we don't care that we do it in each roundtrip? Wouldn't it then be best to call this only when Validate is clicked in the webtests?